Gemfile.lock

```diff
@@ -0,0 +1,251 @@
+PATH
```

Comment:
I added it lazily because Code Climate was unhappy without. Back to the drawing board.